automatic/curl/README.md

```diff
@@ -0,0 +1,7 @@
+# <img src=""></img> [curl](https://chocolatey.com/packages/curl)
```

Comment:
Ah, I see you noticed I had updated the `Update-IconUrl` script to also replace the image url in readme files.
👍